tests/foreman/ui/test_remoteexecution.py

```diff
@@ -0,0 +1,227 @@
+"""Test class for Remote Execution Management UI"""
+from robottelo.decorators import stubbed, tier1, tier2
+from robottelo.test import UITestCase
+
+
+class TestRemoteExecution(UITestCase):
```

Comment:
Not sure what is changed, but I still see `TestRemoteExecution` title when it should be like I mentioned in my previous comment:
`TestRemoteExecution` -> `RemoteExecutionTestCase`